src/Core/src/Handlers/HybridWebView/HybridWebViewHandler.iOS.cs

```diff
@@ -165,7 +165,7 @@ public void StartUrlSchemeTask(WKWebView webView, IWKUrlSchemeTask urlSchemeTask
 
 			private (byte[] ResponseBytes, string ContentType, int StatusCode) GetResponseBytes(string? url)
 			{
-				if (Handler is null)
+				if (Handler is null || !Handler.HasContainer)
```

Comment:
I think the HasContainer may not be the correct property as this is true only when there is some reason we need to wrap the virtual view in some wrapper view to get some feature.

I think your original comment in the issue was correct. We use the `Handler.VirtualView` in this code, but we don't check for null - but we can't do that as accessing the property will throw.

The reason I would think this would happen is that the virtual view got collected somehow while the JS was running. Whatever the reason, you will need to cast to the `IViewHandler` interface and use that to check for the null:

```suggestion
				if (Handler is null || Handler is IViewHandler ivh && ivh.VirtualView is not null)
```